### DISCUSSION TITLE
srt字幕无法下载

### DISCUSSION THREAD

**Discussion by fxb6666** [yutto-dev/yutto#163] [General]
不能下载港澳台番剧的srt字幕了，几天前还是可以的。用bilili也不行。
`yutto -b -n 4 -p 1 https://www.bilibili.com/bangumi/play/ss44484`

**Comment by lc4t** [comment]
这个地址直接访问是404，可能是下架了所以没法下载？

**Reply by fxb6666** [reply]
我能打开啊，这个是仅限港澳台的，要港澳台ip才能打开。

**Reply by lc4t** [reply]
测试了下，获取字幕的API在不登录的情况下不返回字幕地址。解决方式是使用下载时增加：`-c SESSDATA` @fxb6666

**Reply by fxb6666** [reply]
谢谢，按照你的方法，现在可以下载了。只是不知为什么突然要cookie了。
